internal/usage/usage.go

```diff
@@ -428,4 +428,6 @@ dbName and collection are required only for built-in roles.`
 	CompletedAfter                            = "Date filter of which the backup snapshots where completed. Format must be YYYY-MM-DD"
 	Tag                                       = "List that contains key-value pairs between 1 to 255 characters in length for tagging and categorizing the cluster."
 	UpdateWarning                             = " Passing this flag replaces preexisting data."
+	EnableWatch                               = "Watch the command until it completes its execution."
```

Comment:
```suggestion
	EnableWatch                               = "Watch the command until it completes its execution or the watch times out. To set the time that the watch times out, use the --watchTimeout option."
```

Reply:
Thank you so much for the suggestions! Updated :3